CRM/Contact/Page/View/Relationship.php

```diff
@@ -100,7 +100,7 @@ public function view() {
     $rType = CRM_Utils_Array::value('rtype', $viewRelationship[$this->getEntityId()]);
     // add viewed contribution to recent items list
     $url = CRM_Utils_System::url('civicrm/contact/view/rel',
-      "action=view&reset=1&id={$viewRelationship[$this->getEntityId()]['id']}&cid={$this->getContactId()}&context=home"
+      "action=view&reset=1&id={$viewRelationship[$this->getEntityId()]['id']}&context=home"
```

Comment:
Missing `rtype`

Reply:
is rtype needed if we have the relationship id?

Reply:
Ideally no - I think @demeritcowboy is suggesting it doesn't work right without it with the code as currently is?

Reply:
Right - at the moment the way the destination page works, without rtype you get a mostly blank page when you click the recent items view link.